packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -777,6 +783,15 @@ class AndroidCameraCameraX extends CameraPlatform {
     await _unbindUseCaseFromLifecycle(preview!);
   }
 
+  /// Sets the active camera while recording.
+  ///
+  /// Currently unsupported, so is a no-op.
+  @override
+  Future<void> setDescriptionWhileRecording(CameraDescription description) {
```

Comment:
Does this mean we have a feature we know will silently stop working when we modify the default to camerax?

Reply:
It does mean that :/ but the fact that it will be a breaking change can mitigate that fact; it will be noted in the README here but I can also add it to the camera/camera CHANGELOG and camera/camera README.

@stuartmorgan any thoughts on this? I realized recently that this was not implemented for camera_android_camerax.

Reply:
I would definitely note the loss of functionality in the CHANGELOG for the `camera` breaking change.

This is a pretty new feature IIRC, and I would expect relatively niche, so losing it in a breaking change doesn't seem like a major issue. People will always have the possibility of using `camera_android` (or a fork of it, if/when we stop maintaining it) if they need it.